sse2neon.h

```diff
@@ -2177,6 +2232,10 @@ FORCE_INLINE __m128 _mm_or_ps(__m128 a, __m128 b)
 // https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm_prefetch
 FORCE_INLINE void _mm_prefetch(char const *p, int i)
 {
+    (void) i;
+#if defined(_MSC_VER)
+    __prefetch(p);
```

Comment:
This emits `prfm PLDL1KEEP,[x0]`, which seems to be the equivalent of `__builtin_prefetch(p, 0, 3)`. There may be MSVC intrinsics to get the other types of prefetch instructions but I don't think I ever looked into it.

Reply:
Looks like there is a `__prefetch2` https://learn.microsoft.com/en-us/cpp/intrinsics/arm64-intrinsics?view=msvc-170

Although to quote the doc:
```However, from the C++ language point of view, the function has no observable effect, and may do nothing at all.```

So is this worth it at all?

Reply:
__prefetch2 seems to be it. A few macros exist in winnt.h that demonstrate using it for different types of prefetches.